manual/parts/vspace.tex

```diff
@@ -100,8 +99,8 @@ \subsubsection{AArch32}
 
 \subsubsection{AArch64}
 
-Arm AArch64 processors have a four-level page-table structure, where the VSpace is realised as a
-\texttt{PageGlobalDirectory}. All paging structures are index by 9 bits of the virtual address.
+Arm AArch64 processors have a four-level page-table structure. The VSpace object is implemented by the
+\obj{PageGlobalDirectory} object. All paging structures are indexed by 9 bits of the virtual address.
```

Comment:
This is configuration dependent. On some aarch64 hypervisor configurations, the top level object is the PageUpperDirectory and there is no PageGlobalDirectory.  To help with this libsel4 defines the seL4_ARM_VSpaceObject alias that is a preprocessor macro that expands to seL4_ARM_PageUpperDirectoryObject or seL4_ARM_PageGlobalDirectoryObject.

Reply:
Excellent point, will add that. (I had only looked at what was already in the manual)

Reply:
Ok, done. Coincidentally, the reference to PageGlobalDirectory didn't actually exist in the genereated API docs, but is `group__aarch64__seL4__ARM__VSpace` instead. Added that change as well.